system/datatypes/mailinglist.php

```diff
@@ -421,78 +421,80 @@ private function editMailingAction($mailingId = null) {
 		// throw error if edit is requested without client logged in
 		if (!isUser()) return notify('error', __('login-to-edit'));
 
+		$sendMailing = $this->isPosted(self::FORM_CMD_MAILING_SEND);
+		$saveMailing = $this->isPosted(self::FORM_CMD_MAILING_SAVE);
+		if ($this->isPosted() && !$sendMailing && !$saveMailing) {
+			return notify('error', __('ml_invalid-command'));
+		}
+		$isPosted = $sendMailing || $saveMailing;
+
 		// get mailing if id was given
 		/** @var HyphaDomElement $mailing */
-		if ($mailingId) {
+		if (null != $mailingId) {
```

Comment:
Is this needed or better? Implict falseness test seems sufficient here, and I usually find it clear. A != null check might actually behave the same, due to the implicit conversions PHP does (perhaps you intended !== null?).